Package.swift

```diff
@@ -10,12 +28,10 @@ let package = Package(
         .executable(name: "tests", targets: ["PNGTests"]),
         .executable(name: "benchmarks", targets: ["PNGBenchmarks"])
     ],
-    targets:
+    dependencies:
```

Comment:
(Read the following comment about the tests first.)

***

> Why have you added this dependency?

Because I was lazy. All it is used for is providing the simplified [shell functions used to execute the tests](https://github.com/kelvin13/png/pull/12/files#diff-7c0e705781491f42db9d3b4e4c319cdcR10) from inside a traditional test suite. It is not used by any of the products, so [SE‐0226](https://github.com/apple/swift-evolution/blob/master/proposals/0226-package-manager-target-based-dep-resolution.md) means it will not be treated as a dependency by your clients.

If you want, you could use `Foundation`’s more complicated `Process` API directly instead.

Reply:
This is gone now.